metrics/factory.go

```diff
@@ -17,7 +17,11 @@
 
 package metrics
 
-import "time"
+import (
+	"time"
+
+	"github.com/mysteriumnetwork/node/metadata"
```

Comment:
My advice is not to couple `metadata` and `metrics` components, because they can be perfectly uncoupled